src/stratigraphy/depths_materials_column_pairs/depths_materials_column_pairs.py

```diff
@@ -41,3 +41,19 @@ def to_json(self) -> dict:
             ],
             "page": self.page,
         }
+
+    @classmethod
+    def from_json(cls, json_depths_materials_column_pairs: dict) -> "DepthsMaterialsColumnPairs":
+        """Converts a dictionary to an object.
+
+        Args:
+            json_depths_materials_column_pairs (dict): A dictionary representing the depths materials column pairs.
+
+        Returns:
+            DepthsMaterialsColumnPairs: The depths materials column pairs object.
+        """
+        depth_column = DepthColumn.from_json(json_depths_materials_column_pairs["depth_column"])
```

Comment:
How does it know which implementation of the abstract `from_json` method to call here, the one from `BoundaryDepthColumn` or the one from `LayerDepthColumn`?